### PR TITLE
add catalog grid

### PR DESCRIPTION
**Pull request by VladyslavKolisnyk**
DEMO LINK https://VladyslavKolisnyk.github.io/layout_catalog_grid

### CODE REVIEW

File: src/styles/bloks/_catalog.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    align-content: center;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+
+  @media (min-width:488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
```
Comment (vlad-demchuk) [inline src/styles/bloks/_catalog.scss]: move width to variable, you repeat it many times

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/styles/bloks/_catalog.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    border-radius: 5px;
+  }
+
+  @media (min-width:488px) {
+    & {
+      grid-template-columns: repeat(2, $width-card);
+    }
+  }
+
+  @media (min-width:748px) {
```
Comment (anstsot) [inline src/styles/bloks/_catalog.scss]: image
```suggestion
  @media (min-width:768px) {
```

**Review by anstsot — CHANGES_REQUESTED** [review]
It’s almost done! Let’s improve a little

File: src/styles/utils/_variables.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+$width-card: 200px;
```
Comment (volodymyr-soltys97) [inline src/styles/utils/_variables.scss:1]: It is recommended to add a variable for the gap between cards as well.

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,14 +1,4 @@
...
-* {
-  box-sizing: border-box;
-}
-
-@import "_header.scss";
-@import "_catalog.scss";
+@import "./utils/reset";
+@import "./utils/variables";
+@import "./bloks/header";
+@import "./bloks/catalog";
```
Comment (volodymyr-soltys97) [inline src/styles/main.scss:4]: The import path should be './bloks/_catalog.scss' to keep consistent with the task description.

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**